quality/fca/cypress/integration/usager/connexion-secteur-prive.feature

```diff
@@ -1,5 +1,5 @@
 #language: fr
-@usager @ignoreInteg01 @ci
+@ignoreInteg01
```

Comment:
suggestion: could be removed from this PR but ok

Reply:
I was to lazy to put it in a separate PR. There are the last occurences of \@usager and \@ci. I guess because of concurency in merges at the time we droped these.